samples/java-doc-snippets/src/main/java/com/example/DelegatingServiceAction.java

```diff
@@ -0,0 +1,41 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+
+package com.example;
+
+import com.akkaserverless.javasdk.action.ActionCreationContext;
+import com.google.protobuf.Empty;
+
+import java.util.concurrent.CompletionStage;
+
+/** An action. */
+public class DelegatingServiceAction extends AbstractDelegatingServiceAction {
+
+  public DelegatingServiceAction(ActionCreationContext creationContext) {}
+
+  // tag::delegating-action[]
+  @Override
+  public Effect<DelegatingServiceApi.Result> addAndReturn(DelegatingServiceApi.Request request) {
+    CounterServiceClient counterClient = actionContext().getGrpcClient(CounterServiceClient.class, "counter"); // <1>
+
+    CounterApi.IncreaseValue increaseValue = CounterApi.IncreaseValue.newBuilder()
+        .setCounterId(request.getCounterId())
+        .setValue(1)
+        .build();
+    CompletionStage<Empty> increaseCompleted = counterClient.increase(increaseValue);  // <2>
+
+    CompletionStage<CounterApi.CurrentCounter> currentCounterValueAfter = increaseCompleted.thenCompose((__) -> // <3>
```

Comment:
that `(__)` looks a bit fancy (foreign)? `notUsed ->` instead?

Reply:
I thought `__` was the canonical way to mark not used parameters in Java land, but sure.

Reply:
if that's the way it's done by others (not Scala ppl) I'm fine with that

Reply:
I can't find any reference to it on the line, so maybe I misremember (`_` did become a keyword in Java 9 though, so maybe I heard some JVM talk about future plans for that)